src/screens/Page.js

```diff
@@ -0,0 +1,265 @@
+          </PropertyValue>
+        </Property>
+
+        <Property name="page.narrow.large">
+          <Description>Any valid Box props.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>{`{ pad: { horizontal: 'large' }}`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="page.full.alignSelf">
+          <Description>How to align the content on a Page.</Description>
+          <PropertyValue type="string">
+            <Example defaultValue>"start"</Example>
+            <Example>"center"</Example>
+            <Example>"end"</Example>
+            <Example>"stretch"</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="page.full.width">
+          <Description>The min and max width of the content.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>{`{ min: "medium", max: "100%" }`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="page.full.small">
+          <Description>Any valid Box props.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>{`{ pad: { horizontal: 'large' }}`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="page.full.medium">
+          <Description>Any valid Box props.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>{`{ pad: { horizontal: 'medium' }}`}</Example>
+          </PropertyValue>
+        </Property>
+
+        <Property name="page.full.large">
+          <Description>Any valid Box props.</Description>
+          <PropertyValue type="object">
+            <Example defaultValue>{`{ pad: { horizontal: 'large' }}`}</Example>
+          </PropertyValue>
+        </Property>
+      </ThemeDoc>
+    </ComponentDoc>
+
+    <ComponentDoc
+      name="PageContent"
+      description="An container for content within a Page"
+      intrinsicElement="div"
+      isA={{
+        base: 'Box',
+        path: '/box',
+        defaultProps: {
+          fill: 'horizontal',
+        },
+      }}
+    >
+      <Properties>
+        <Property name="background">
+          <Description>
+            The background of the Page. Use `full: "horizontal"` to specify that
+            the background should ignore width restrictions and stretch across
+            the full width of the Page.
+          </Description>
+          <PropertyValue type="string">
+            <Description>A color or image url.</Description>
+            <Example>"neutral-1"</Example>
+            <Example>"url(//my.com/assets/img.png)"</Example>
+          </PropertyValue>
+          <PropertyValue type="object">
+            <Example>
+              {`
+{
+  color: "neutral-1",
+  dark: true,
+  opacity: true,
+  position: "bottom",
+  repeat: "no-repeat",
+  size: "cover",
+  image: "url(//my.com/assets/img.png)",
+  full: "horizontal
```

Comment:
```suggestion
  fill: "horizontal
```

Reply:
Thanks for catching these! Going back and fixing...